setup

```diff
@@ -95,19 +95,24 @@ storage() {
 	mkdir -p "${ROOTFS}/var/lib/shared/overlay-images" \
 	      "${ROOTFS}/var/lib/shared/overlay-layers" \
               "${ROOTFS}/var/${TMP_QM_IMG_DIR}"
-	touch "${ROOTFS}/var/lib/shared/overlay-images/images.lock" \
+
+        touch "${ROOTFS}/var/lib/shared/overlay-images/images.lock" \
 	      "${ROOTFS}/var/lib/shared/overlay-layers/layers.lock"
 
         sed -e '/additionalimage.*/,/]/s/^#//g' \
             -e '/additionalimages.*/{n;s/.*/\"\/var\/lib\/shared\",/;}' \
             -e 's|^#.*transient_store.*|transient_store=true|g' \
             /"${ROOTFS}/usr/share/containers/storage.conf" \
             > "${ROOTFS}/etc/containers/storage.conf"
+    fi
 
-        if ! grep -q "env" "${ROOTFS}/etc/containers/containers.conf"; then
-            echo "env = [\"TMPDIR=/var/${TMP_QM_IMG_DIR}\"]" \
-            >>  "${ROOTFS}/etc/containers/containers.conf"
-        fi
+    #File always copied from /usr/share/qm, check if exist and append
```

Comment:
just a question, what happens if the file doesn't exists ?

Reply:
Containers are too big, from time to the default /var/tmp
Got this error while pulling an image 
Error: writing blob: adding layer with blob "sha256:b45265b317a7897670ff015b177bac7b9d5037b3cfb490d3567da959c7e2cf71": Error processing tar file(exit status 1): write /a65be6ac39ddadfec332b73d772c49d5f1b4fffbe7a3a419d00fd58fcb4bb757/layer.tar: no space left on device

Now in this PR that we have /var partition instead of /var/qm could be that it is not needed, 
I can open a ticket to investigate it, please share your thoughts here

Reply:
It's not the first time we have complains about container too big, if it's something really hard to us. Let's reduce but on the other hand, if we have that big, we might reaching a different scenario in our tests with additional spaces/partitions etc. Yes, please open a ticket for that @Yarboa

Reply:
Sure, done
#295